server/views/pages/goal/review/index.njk

```diff
@@ -28,7 +28,7 @@
                 Description
               </dt>
               <dd class="govuk-summary-list__value" data-qa="goal-{{ goalLoop.index }}-description-value">
-                {{ goal.title }}
+                {{ goal.title | formatMultilineText | safe }}
```

Comment:
The only gotcha is that we need to also use the `safe` filter, because the text now has HTML in it (ie. the `<br />` from the filter)